Cards/Resource/Card_Debuff_healing.tres

```diff
@@ -0,0 +1,48 @@
+[gd_resource type="Resource" script_class="CardBase" load_steps=14 format=3 uid="uid://bek3vbtsf3x2l"]
+
+[ext_resource type="Script" path="res://Cards/Effects/EffectApplyStatus.gd" id="1_63tqs"]
+[ext_resource type="Script" path="res://Status/Debuffs/Debuff_healing.gd" id="2_7i6kf"]
+[ext_resource type="Script" path="res://Entity/Components/Stats/StatModifiers.gd" id="3_omw4j"]
+[ext_resource type="Script" path="res://Cards/Effects/EffectData.gd" id="4_hb64u"]
+[ext_resource type="Script" path="res://Cards/CardBase.gd" id="5_8c6ak"]
+[ext_resource type="Script" path="res://Cards/Targeting/TargetPlayer.gd" id="5_u0kut"]
```

Comment:
if the card is targeting all then the targeting function shouldn't be target player but target base